### PR TITLE
Add support for NEO Flex and Vortex

### PR DESCRIPTION
**Pull request by brettle**
Fixex #65 and #66.

### CODE REVIEW

File: plugin/controller/src/webotsFolder/dist/protos/deepbluesim/PreconfiguredDBSLinearMotor.proto [hunk 1]
```diff
@@ -18,8 +18,10 @@ PROTO PreconfiguredDBSLinearMotor [
 {
     %<
         import {assert} from 'wbutility.js';
-        assert(!fields.motorType.value.startsWith("NEO") || ["Spark Max", "PWM"].includes(fields.controllerType.value),
-            `PreconfiguredDCDBSMotor: Brushless motor "${fields.motorType.value}" cannot be driven by brushed motor controller "${fields.controllerType.value}"!`)
+        assert(!fields.motorType.value.startsWith("NEO") || ["Spark Max", "Spark Flex", "PWM"].includes(fields.controllerType.value),
+            `PreconfiguredDCDBSMotor: Brushless motor "${fields.motorType.value}" cannot be driven by brushed motor controller "${fields.controllerType.value}"!`);
+        assert(fields.controllerType.value !== "NEO Vortex" || ["NEOVortex"].includes(fields.motorType.value),
```
Comment (CoolSpy3) [inline plugin/controller/src/webotsFolder/dist/protos/deepbluesim/PreconfiguredDBSLinearMotor.proto:23]: No need to use `includes` when comparing against a single value
```suggestion
        assert(fields.controllerType.value !== "NEO Vortex" || fields.motorType.value === "NEOVortex",
```

File: plugin/controller/src/webotsFolder/dist/protos/deepbluesim/PreconfiguredDBSRotationalMotor.proto [hunk 2]
```diff
@@ -17,8 +17,10 @@ PROTO PreconfiguredDBSRotationalMotor [
 {
     %<
         import {assert} from 'wbutility.js';
-        assert(!fields.motorType.value.startsWith("NEO") || ["Spark Max", "PWM"].includes(fields.controllerType.value),
-            `PreconfiguredDBSRotationalMotor: Brushless motor "${fields.motorType.value}" cannot be driven by brushed motor controller "${fields.controllerType.value}"!`)
+        assert(!fields.motorType.value.startsWith("NEO") || ["Spark Max", "Spark Flex", "PWM"].includes(fields.controllerType.value),
+            `PreconfiguredDCDBSMotor: Brushless motor "${fields.motorType.value}" cannot be driven by brushed motor controller "${fields.controllerType.value}"!`);
+        assert(fields.controllerType.value !== "NEO Vortex" || ["NEOVortex"].includes(fields.motorType.value),
```
Comment (CoolSpy3) [inline plugin/controller/src/webotsFolder/dist/protos/deepbluesim/PreconfiguredDBSRotationalMotor.proto:22]:
```suggestion
        assert(fields.controllerType.value !== "NEO Vortex" || fields.motorType.value === "NEOVortex",
```

**Review by CoolSpy3 — CHANGES_REQUESTED** [review]

File: plugin/controller/src/webotsFolder/dist/protos/deepbluesim/PreconfiguredDBSLinearMotor.proto (hunk 1 above)
Comment (brettle) [reply plugin/controller/src/webotsFolder/dist/protos/deepbluesim/PreconfiguredDBSLinearMotor.proto]: This was intentional to make it easier if additional motors are released that the Spark Flex can drive.

File: plugin/controller/src/webotsFolder/dist/protos/deepbluesim/PreconfiguredDBSRotationalMotor.proto (hunk 2 above)
Comment (brettle) [reply plugin/controller/src/webotsFolder/dist/protos/deepbluesim/PreconfiguredDBSRotationalMotor.proto]: This was intentional to make it easier if additional motors are released that the Spark Flex can drive.

**Review by CoolSpy3 — APPROVED** [review]

**Pull request merged**